### ISSUE TITLE
Collect feedback and ideas for the project logo

### ISSUE THREAD

**Issue by FelixHenninger** [FelixHenninger/lab.js#16]
Our project logo has served us well, and we still enjoy it, but it's clear that there's potential to do better. We chose the flask icon when `lab.js` was still very much an experiment we were brewing, but our project is much bigger and more stable now. **It would be great to work toward an adjusted or updated logo that more clearly reflects what the project is about** -- our users are interested in the mind more than chemistry. 

**Any thoughts or ideas are very much welcome!**

![labjs-mozillapulse](https://user-images.githubusercontent.com/8349942/39156088-fc044872-4754-11e8-80a7-4ee6a74ee20a.png)

**Comment by chartgerink** [comment]
I browsed [the Noun project](https://thenounproject.com/) for icons you could use and found a few suggestions :smile_cat: 

![noun_1361937](https://user-images.githubusercontent.com/2946344/39870653-6e4cabda-5462-11e8-88ed-31ec6178e15a.png)
![noun_1326883](https://user-images.githubusercontent.com/2946344/39870654-6e8f631c-5462-11e8-99a8-3d5b16c67a54.png)
![noun_1454290](https://user-images.githubusercontent.com/2946344/39870655-6ece3e5c-5462-11e8-9201-c831e71ee479.png)
![noun_1111386](https://user-images.githubusercontent.com/2946344/39870656-6f02b7ea-5462-11e8-81af-c671b9cf2f73.png)
![noun_348832](https://user-images.githubusercontent.com/2946344/39870657-6f366d7e-5462-11e8-9c73-6c95fff76fd3.png)
![noun_1361958](https://user-images.githubusercontent.com/2946344/39870658-6f973a64-5462-11e8-9d8a-5b6b214b6a19.png)

**Comment by FelixHenninger** [comment]
@chartgerink Awesome, thanks! I see we've taken [the same approach](https://thenounproject.com/term/flask/123666/) 😉. I really like the idea of the brain being stuck in some kind of container, maybe it would be cool to replace the fluid with a more brainy-style substance?

**Comment by Baddaboo** [comment]
That fourth one reminds of of Cambridge Analytica. Perhaps not the best connotation 😂

![logo](https://upload.wikimedia.org/wikipedia/en/b/b9/Cambridge_Analytica_logo.png)

**Comment by FelixHenninger** [comment]
Awesome, thanks, nice catch! I totally missed that one 😂 Would certainly have been rather unfortunate 😅

**Comment by vsoch** [comment]
@FelixHenninger I love the logo! My 0.02 is that the colors and simplicity are right on, and I wouldn't move in a different direction or try to change it.

**Comment by FelixHenninger** [comment]
Wow, thanks for the feedback!

I'm kinda split, I like it too but I'm probably biased -- we got some mixed feedback out of Mozilla's open leadership cohort this spring, and I see their points, so it's been at the back of my mind -- but then again I haven't had any better ideas. It probably reflects my physics/chemistry envy more than anything else 😉 .

I made a low-resolution version recently because the original didn't work at small sizes, but after playing around with it for a while, found that it also works on medium sizes (3" across). Do you think it's better?

![logo_yellow_lowres](https://user-images.githubusercontent.com/8349942/42426905-fb715d7e-832b-11e8-84cb-81487aaccb6d.png)

**Comment by vsoch** [comment]
Yes, it's beautiful and simple and communicates the name really well.

**Comment by FelixHenninger** [comment]
Ok, thanks! I'm going to leave this issue open a bit longer for further feedback, but I'm much more confident thanks to your comments.
